2022/_data/jobs.yml

```diff
@@ -374,5 +374,109 @@
+        "link": "https://www.youtube.com/watch?v=-4digbYIFxs&list=PLKrSWEllR0Lg5c0eYT5AEi0vamlAZHilb"
+      },
+      {
+        "title": "System Engineer - Internal IT",
+        "location": "Remote/ Onsite BCN or Vigo",
+        "description": "It is all about Systems! As a System Engineer, you make sure that these systems run consistently stable and well. You develop software to automate or to integrate systems. You are also involved in designing the infrastructure for the solution and developing the system engineering aspects. The jobs is very diverse, from coding excellent automations to client communication and occasionally saving our clients world.<br/><br/>You are a good fit if you consider yourself as a team player who is looking forward to operating and constantly improving secure and highly performing Systems.<br/><br/><strong>You will not be bored, as your role involves the following:</strong><br/>- Collaborate with team members to improve the company’s engineering tools, systems and procedures, and data security<br/>- Analyze code and discuss with the teams to ensure a marked improvement in applications and the timely completion of projects<br/>- Building and maintaining the technology stack conforming to the documented architecture blueprint<br/><br/><strong>What we require:</strong><br/>- Substantial senior experience to be able to do the job with your eyes closed (well, almost)<br/>- Fluent in English<br/>- Strong focus on Automation<br/>- AWS administration, Google Cloud or Azure<br/>- Linux administration: CentOS or RedHat<br/>- Scripting: Bash, Ruby or Python<br/>- Configuration Management: Puppet, Ansible or Chef<br/>- Provisioning tools: Foreman, Salt or Terraform<br/>- Continuous Integration: Jenkins, Nexus or Sonar<br/>- Monitoring tools: Nagios, Zabbix, Prometheus or Sensu<br/>",
```

Comment:
I just pass text to a docs spell checker and found

* Missing coma after `consistently`.
* `The jobs` should be singular based on context and rest of sentence.
* `saving our clients world` should be `clients` ?

Reply:
I don't know what they mean with _saving our clients world_. I think we should just copy and paste and then ask them to review. I'd rather remove that piece of text

Reply:
You are right, in case of doubt let's leave as it is. I woudn't not bother checking further given timings.